openedx/features/edly/context_processor.py

```diff
@@ -0,0 +1,267 @@
+from math import floor
+
+from django.core.cache import cache
+
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+
+
+CACHE_NAME = 'context_processor.dynamic_theming'
+CACHE_TIMEOUT = 60
+DEFAULT_COLOR_DICT = {
+    'primary': '#3E99D4',
+    'secondary': '#1197EA'
+}
+DEFAULT_FONTS_DICT = {
+    'base-font': "'Open Sans', sans-serif",
+    'heading-font': "'Open Sans', sans-serif",
+    'font-path': "<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,600,700&display=swap' rel='stylesheet' />",
```

Comment:
This should be only `https://fonts.googleapis.com/css?family=Open+Sans:400,600,700&display=swap`
CC: @mamjad00

Reply:
i have fixed it